netjsonconfig/backends/airos/wpasupplicant.py

```diff
@@ -0,0 +1,152 @@
+from interface import encryption, ssid
```

Comment:
I think we need a relative import here because I'm getting an exception while testing